xapian-glib/xapian-utils.cc

```diff
@@ -135,28 +137,40 @@ xapian_revision (void)
 /**
  * xapian_sortable_serialise:
  * @value: The numeric value to serialize
+ * @len: (out): Used to return the length of the returned string
  *
- * Convert a floating point number to a string, preserving sort order.
+ * Convert a floating point number to a string, preserving sort order. The
+ * returned string may contain embedded zero bytes, so its length is also
+ * returned.
  *
  * This method converts a floating point number to a string, suitable for using
  * as a value for numeric range restriction, or for use as a sort key.
  *
- * Returns: (transfer full) (array zero-terminated=1) (element-type guint8): the serialised value
+ * Returns: (transfer full) (array length=len): the serialised value
  *
  * Since: 1.4
  */
-char *
-xapian_sortable_serialise (double value)
+guchar *
+xapian_sortable_serialise (double value, gsize *len)
```

Comment:
Coding style: arguments after the first go on a new line, aligned with the first argument.